README_extending_docker.md

```diff
@@ -93,4 +93,48 @@ Lets consider two scenarios as starting point for extending Awesome docker
 
 ### How to create Custom Actions
 
-You can refer to [this link](https://docs.unskript.com/unskript-product-documentation/actions/create-custom-actions) on how to create custom Action
+You can create custom action on your workstation using your editor. Please follow the steps below to setup your workstation:
+
```

Comment:
You can also refer them to https://realpython.com/intro-to-pyenv/#:~:text=Activating%20Your%20Versions,-Now%20that%20you&text=python%2Dversion%20file%20in%20your,environment%20will%20automatically%20be%20activated. if they want to use pvenv

Reply:
I dont want to document multiple ways, if the user knows it can be accomplished via pyenv, let them use that.